CRM/Civicase/Hook/CaseCategoryInstanceBase.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Class CRM_Civicase_Hook_CaseCategoryInstanceBase.
+ * Base class for the CaseCategoryInstance hook classes.
```

Comment:
I think it will be good if we can add more details of the functionality of the class